proto/metadata.proto

```diff
@@ -298,4 +298,15 @@ message Metadata {
   // The `platform` field for each `platform_exceptions` should be mutually
   // exclusive. Duplicate matches will result in a test failure.
   repeated PlatformExceptions platform_exceptions = 5;
+
+  enum Tags {
+    AGGREGATION = 0;
+    DATACENTER_EDGE = 1;
+    EDGE = 2;
+    TRANSIT = 3;
+  }
+
+  // The `tags` used to identify the area(s) testcase applies to. An empty tag
+  // is the default implying it applies to all areas.
```

Comment:
Also run `make proto/metadata_go_proto/metadata.pb.go` to generate the pb.go files